network/src/inbound/inbound.rs

```diff
@@ -192,8 +119,6 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
             unreachable!("All messages processed sent to the inbound receiver are Inbound");
         };
 
-        self.peer_book.register_message(source);
```

Comment:
why is this removed? it updates the number of messages received from the specific peer and its last activity timestamp

Reply:
I see it's been moved into the peer code; the diff was just collapsed